### ISSUE TITLE
Export chat modal has white text on a white background

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#19238]
### Steps to reproduce

Open the "Export chat" modal

### What happened?

![export](https://user-images.githubusercontent.com/48614497/135478369-86fc9c45-2f1b-4303-95f5-e25c133d44b5.png)
![selected](https://user-images.githubusercontent.com/48614497/135478381-f04a27e7-56a3-457d-9270-a3cd14bb4ef7.png)

The modal has text that cannot be seen in light mode because it is the same color as the background.

### What did you expect?

The text should be readable in both light and dark modes.

### Operating system

NixOS unstable

### Browser information

Firefox 92.0

### URL for webapp

develop.element.io

### Homeserver

townsendandsmith.ml

### Have you submitted a rageshake?

No
